azkaban-common/src/main/java/azkaban/jobcallback/JobCallbackConstants.java

```diff
@@ -0,0 +1,36 @@
+package azkaban.jobcallback;
+
+public interface JobCallbackConstants {
+  public static final String STATUS_TOKEN = "status";
+  public static final String SEQUENCE_TOKEN = "sequence";
+  public static final String HTTP_GET = "GET";
+  public static final String HTTP_POST = "POST";
+
+  public static final String MAX_CALLBACK_COUNT_PROPERTY_KEY =
+      "jobcallback.max_count";
+  public static final int DEFAULT_MAX_CALLBACK_COUNT = 3;
+
+  public static final String FIRST_JOB_CALLBACK_URL_TEMPLATE =
+      "job.notification." + STATUS_TOKEN + ".1.url";
+
+  public static final String JOB_CALLBACK_URL_TEMPLATE = "job.notification."
```

Comment:
I think it is better to provide a function here so that STATUS_TOKEN and SEQUENCE_TOKEN can be replaced in one call.